bluebinder.c

```diff
@@ -176,6 +214,60 @@ dev_write_packet(
     }
 }
 
+static
+void
+configure_bt(
+    struct proxy *proxy,
+    gboolean bluetooth_on)
+{
+    int status = 0;
+    bool fail = FALSE;
+
+    if (bluetooth_on && !proxy->bluetooth_on) {
+        GBinderRemoteReply *reply;
+        GBinderLocalRequest *initialize_request;
+
+        fprintf(stderr, "Turning bluetooth on\n");
+
+        initialize_request = gbinder_client_new_request(proxy->binder_client);
+
+        gbinder_local_request_append_local_object
+            (initialize_request, proxy->local_callbacks_object);
+
+        reply = gbinder_client_transact_sync_reply
+            (proxy->binder_client, 1 /* initialize */, initialize_request, &status);
```

Comment:
I'd use a enum to define transaction codes, but that would work too. Feel free to ignore.